src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -568,6 +568,18 @@ public function find($id, $columns = ['*'])
         return $this->whereKey($id)->first($columns);
     }
 
+    /**
+     * Find a model once by its primary key.
+     *
+     * @param  mixed  $id
+     * @param  array|string  $columns
+     * @return ($id is (\Illuminate\Contracts\Support\Arrayable<array-key, mixed>|array<mixed>) ? \Illuminate\Database\Eloquent\Collection<int, TModel> : TModel|null)
```

Comment:
Shouldn't `find()` always return `TModel`? For `\Illuminate\Database\Eloquent\Collection<int, TModel>`, we have `get()`

Reply:
We can pass an array/collection of IDs to fetch multiple records, though (it’s also mentioned in the original phpDoc of `find` and `findOrFail`)

But same here - I also always use `find` for fetching a single record and `get()` for a list 😄

Reply:
But what was the decision to not have a `getOnce()`?